WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -352,3 +353,33 @@ private void SetAutoCoinJoin(bool enabled)
 
 	public ICommand StopCommand { get; }
 }
+
+internal class AmountViewModel : RemainingPartViewModel
+{
+	public AmountViewModel(Money amount)
+	{
+		Text = amount.ToFormattedString() + " BTC";
+	}
+
+	public string Text { get; }
+}
+
+internal class EmptyViewModel : RemainingPartViewModel
+{
+}
+
+public abstract class RemainingPartViewModel
+{
+}
+
+internal class RemainingTimeViewModel : RemainingPartViewModel
+{
+	public TimeSpan Time { get; }
+
+	public RemainingTimeViewModel(TimeSpan time)
+	{
+		Time = time;
+	}
+
+	public string Text => $"-{Time.ToString(CoinJoinStateViewModel.TimespanFormat)}";
+}
```

Comment:
I'd like to know here to put all those. Nested classes don't work with DataTemplate's `DataType`, but these are quite specific to their "parent"